src/gui/scaleChargesDialogFuncs.cpp

```diff
@@ -1,21 +1,19 @@
 // SPDX-License-Identifier: GPL-3.0-or-later
 // Copyright (c) 2024 Team Dissolve and contributors
 
+#include "gui/models/scaleChargesDialogModel.h"
 #include "gui/scaleChargesDialog.h"
+#include <QQmlContext>
+#include <QQuickItem>
+#include <QQuickWidget>
```

Comment:
```suggestion
#include <QQuickWidget>
#include <QHBoxLayout>
```

Includes needed for resizing.